apps/dialtone-documentation/docs/about/whats-new/posts/2024-7-8.md

```diff
@@ -8,7 +8,7 @@ posted: '2024-7-8'
 
 **👋 Hey there,** we're excited to share some updates with you:
 
-* We redesigned our [Tokens Page](/tokens) to include a new search feature and the ability to copy tokens easily.
+* We redesigned our [Tokens Page](/tokens/) to include a new search feature and the ability to copy tokens easily.
```

Comment:
All these types of links will link to the main site when under a subdirectory ex if you are on `https://dialtone.dialpad.com/deploy-previews/pr-572` and click the link it will link to `https://dialtone.dialpad.com/tokens`

Is it possible to use withBase in this context? I know it is possible in vue components but maybe not in markdown...

https://vuepress.vuejs.org/guide/assets.html#base-helper

not THAT big of deal but could cause confusion in some cases.